triggers/service/BUILD.bazel

```diff
@@ -73,10 +73,7 @@ da_scala_binary(
 da_scala_test(
     name = "tests",
     srcs = glob(["src/test/scala/**/*.scala"]),
-    data = [
-        ":test-model.dar",
-        "@toxiproxy_nix//:bin/toxiproxy-cmd",
-    ],
+    data = ([":test-model.dar"] + (["@toxiproxy_nix//:bin/toxiproxy-cmd"] if not is_windows else [])),
```

Comment:
This doesn’t work. You have to set it up as a `dev_env_tool` and then change how you locate runfiles to get it from there.

Reply:
Oh. Ok.